.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to `testcontainers-python`
+
+Welcome to the `testcontainers-python` community!
+This should give you an idea about how we build, test and release `testcontainers-python`!
+
+Highly recommended to read this document thoroughly to understand what we're working on right now
+and what our priorities are before you are trying to contribute something.
+
+This will greatly increase your chances of getting prompt replies as the maintainers are volunteers themselves.
+
+## Before you Begin
+
+We recommend following these steps:
+
+1. Finish readng this document.
+2. Read the [recently updated issues][1]
+3. Look for existing issues on the subject you are interested in - we do our best to label everything correctly
+
+
+## Local Development
+
+### Pre-Requisites
+
+You need to have the following tools available to you:
+- `make` - You'll need a GNU Make for common developer activities
+- `poetry` - This is the primary package manager for the project
+- `pyenv` **Recommended**: For installing python versions for your system.
+  Poetry infers the current latest version from what it can find on the `PATH` so you are still fine if you don't use `pyenv`.
+
+### Build and test
+
+
+- Run `make install` to get `poetry` to install all dependencies and set up `pre-commit`
```

Comment:
If someone is not wanting to use pre-commit, there could also be a 'lint' makefile target which runs it on all files

```Makefile
pre-commit-all:
    poetry run pre-commit run --all-files
```
(replace 4 spaces with tab...)

Reply:
done, but a few lines below your comment